onboarding/mlsys/1-gradients.md

```diff
@@ -0,0 +1,494 @@
+# Part 1: Calculating Gradients and Averaging Them Together
+
+Machine learning is all about getting computers to solve math problems for you.
+
+Specifically, it's about solving math problems like, "For a given mathematical
+model, what weights in the model would cause the model to calculate the most
+accurate predictions for a set of inputs?"
+
+You're going to write code to solve this exact problem for one particularly
+simple model.
```

Comment:
In this exercise, you'll write code to solve this exact problem and for one particularly simple model.